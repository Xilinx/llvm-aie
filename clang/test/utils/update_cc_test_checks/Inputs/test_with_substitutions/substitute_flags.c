// RUN: %clang_cc1 %s %ccflags -o - | FileCheck %s

int foo() {
  return 0;
}
