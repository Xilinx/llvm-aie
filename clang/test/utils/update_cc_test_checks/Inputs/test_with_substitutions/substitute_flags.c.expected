// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py
// RUN: %clang_cc1 %s %ccflags -o - | FileCheck %s

// CHECK-LABEL: @foo(
// CHECK-NEXT:  entry:
// CHECK-NEXT:    ret i32 0
//
int foo() {
  return 0;
}
